app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ActiveRecord::Base
+  has_secure_password validations: false
+
+  validates :password, length: {minimum: 3}
+  validates :email, length: {minimum: 5}
```

Comment:
Can I combine these lines even when the number of characters i am enforcing are different?